tests/cmds/test_wallet_funcs.py

```diff
@@ -0,0 +1,307 @@
+from __future__ import annotations
+
+import contextlib
+import io
+import sys
+from decimal import Decimal
+
+import click.testing
+import pytest
+
+from chia.cmds.wallet_funcs import (
+    add_uri_to_nft,
```

Comment:
I'm not sure whether this should also include tests for vanilla spends, and VC spends?